app/javascript/components/miq-about-modal.jsx

```diff
@@ -15,9 +15,14 @@ const showModal = () => (dispatch, getState) => {
   }
 
   // Request the modal data from the API
-  return API.get('/api')
-    .then(data => dispatch({ type: LOAD_ABOUT_MODAL, data }))
-    .then(() => dispatch({ type: SHOW_ABOUT_MODAL }));
+  return API.get('/api').then(root =>
+    API.get(root.server_info.region_href).then(region =>
+      API.get(root.server_info.zone_href).then((zone) => {
+        const data = { ...root };
+        data.region = region;
+        data.zone = zone;
```

Comment:
You can make this a bit nicer :wink:.

```suggestion javascript
        const data = { ...root, region, zone };
```